simtools/applications/simulate_prod.py

```diff
@@ -120,80 +101,6 @@ def _parse(description=None):
 
     """
     config = configurator.Configurator(description=description)
-    config.parser.add_argument(
-        "--production_config",
-        help=(
-            "Simulation configuration file "
-            "(contains the default setup which can be overwritten by the command line options)"
-        ),
-        type=str,
-        required=True,
-    )
-    config.parser.add_argument(
-        "--simulation_software",
-        help="Simulation software steps.",
-        type=str,
-        choices=["corsika", "simtel", "corsika_simtel"],
-        required=True,
-        default="corsika_simtel",
-    )
-    config.parser.add_argument(
-        "--primary",
-        help="Primary particle to simulate.",
-        type=str.lower,
-        required=True,
-        choices=[
-            "gamma",
-            "gamma_diffuse",
-            "electron",
-            "proton",
-            "muon",
-            "helium",
-            "nitrogen",
-            "silicon",
-            "iron",
-        ],
-    )
-    config.parser.add_argument(
-        "--azimuth_angle",
-        help=(
-            "Telescope pointing direction in azimuth. "
-            "It can be in degrees between 0 and 360 or one of north, south, east or west "
-            "(case insensitive). Note that North is 0 degrees and "
-            "the azimuth grows clockwise, so East is 90 degrees."
-        ),
-        type=CommandLineParser.azimuth_angle,
-        required=True,
-    )
-    config.parser.add_argument(
-        "--zenith_angle",
-        help="Zenith angle in degrees (between 0 and 180).",
-        type=CommandLineParser.zenith_angle,
-        required=True,
-    )
-    config.parser.add_argument(
-        "--nshow",
-        help="Number of showers to simulate.",
-        type=int,
-        required=False,
-    )
-    config.parser.add_argument(
-        "--start_run",
-        help=(
-            "Start run number such that the actual run number will be 'start_run' + 'run'. "
-            "This is useful in case a new transform is submitted for the same production. "
-            "It allows the transformation system to keep using sequential run numbers without "
-            "repetition."
-        ),
-        type=int,
-        required=True,
-    )
-    config.parser.add_argument(
-        "--run",
-        help="Run number (actual run number will be 'start_run' + 'run')",
-        type=int,
-        required=True,
-    )
     config.parser.add_argument(
```

Comment:
All the removed arguments still appear in the docstring above. Why did you remove the `config.parser.add_argument`s?

Reply:
No, they are moved to the command_line module, as we use this type of parameters for several applications. So the docstring is still fine (as e.g., you want to configure simulate_prod with azimuth or number of runs).

Reply:
In the docstring the parameters are under command_line_parameters. Maybe you can make clear that the values come from the config file.

Reply:
This is correct, they are command line parameters:

- any argument defined through Configurator can be set on the command line, by environmental variable or using the `--config` option and given them all in a configuration file.

This allows us to have let the user decide to configure tools with super long command lines or using a configuration tool.

Reply:
I would say they are converted into command line parameters if they are passed via a configuration file? Maybe you can add this sentence at the beginning somewhere.

Reply:
This has been changed, as discussed in the chat.